Random-Color-Generator/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<link rel="stylesheet" href="style.css">
+<body>
+    <h1>Random color generator</h1>
+
+    <div class="container">
+        <div>
+          <input type="color" id="color">
+          <h4 id="text">#000000</h4>
+          <p>click to change color</p>
+        </div>
+      </div>
```

Comment:
@TusharRaval 
No README.md provided. No screenshots attached.
No live link to it.
A very laid back approach to submission. Next time it will be completely rejected. Beware!
@NitkarshChourasia